integration-cli/docker_cli_run_test.go

```diff
@@ -3363,7 +3363,7 @@ func TestRunRestartMaxRetries(t *testing.T) {
 		t.Fatal(string(out), err)
 	}
 	id := strings.TrimSpace(string(out))
-	if err := waitInspect(id, "{{ .State.Restarting }} {{ .State.Running }}", "false false", 5); err != nil {
+	if err := waitInspect(id, "{{ .State.Restarting }} {{ .State.Running }}", "false false", 10); err != nil {
```

Comment:
I think it is okay. It won't hurt fast machines, but your laptop should pass tests too.